recipe/meta.yaml

```diff
@@ -34,6 +33,8 @@ requirements:
   run:
     - python
     - asn1crypto >0.24.0,<2.0.0
+    - boto3 >=1.0
+    - botocore >=1.0
```

Comment:
Can you update the PR's initial comment so that URLs point to v3.14.1 instead of current v3.14.0, see https://github.com/snowflakedb/snowflake-connector-python/commit/cb574b6efaa791fc9fd88ca13b93c2352c87a46d